datalad_dataverse/remote.py

```diff
@@ -50,8 +53,31 @@ def transfer_retrieve(self, key, file):
         pass
 
     def remove(self, key):
-        raise
-        pass
+         # connect to dataverse instance
+        api = NativeApi(base_url=self.annex.getconfig('url'),
+                        api_token=os.environ["DATAVERSE_API_TOKEN"])
+        
+        # get the dataset and a list of all files
+        dataset = api.get_dataset(identifier=self.annex.getconfig('doi'))
```

Comment:
`dataset.raise_for_status()` right after that, would allow to properly communicate all kinds of connection errors upwards.